### ISSUE TITLE
NBTEdit not working on servers [Help Wanted]

### ISSUE THREAD

**Issue by GreeningSiren** [MeowCraftMC/NBTEdit#42]
Hello,
I'm using NBTEdit on a server and noticed that both keybind (N) and the command /nbtedit aren't available for use on a minecraft server. I am an OP on the server and would like to be able to use the mod to improve my experience with NBT editing.
Any help is appreciated!
- GreeningSiren

**Comment by qyl27** [comment]
Hello, please provide the version of game and the version of NBTEdit.  
And if you have some logs, things will be better.

**Comment by GreeningSiren** [comment]
Hello,
I'm using 1.21 with version 5.2.7 from Modrinth. Logs won't be in use as nothing is said from the mod when trying to use the features

**Comment by qyl27** [comment]
Have you installed NBTEdit on server?  
It can't work without the server side installed.

**Comment by GreeningSiren** [comment]
Why can it not? The server is using Paper with only support for plugins. Is there a plugin version of NBTEdit?

**Comment by qyl27** [comment]
Because the NBT-saving operation should run on the server.  
There is no plugin version available yet, it is possible but needs more work.  
I had opened an issue about it in #12 last year, but I'm too busy IRL to do that.

**Comment by GreeningSiren** [comment]
Is it possible as a setting to change the use to /data modify commands on the client side so no server mod is needed

**Comment by qyl27** [comment]
It might be another way to implement the feature, but it is also unavailable for now.  
And this way is more difficult than server-side plugin btw.

**Comment by nooksmr** [comment]
I landed here because I though it could replace the heavy loss of [IBE Editor](https://github.com/skyecodes/IBE-Editor) (which have dropped support since October 31st 2024). And I only noticed after that this is a fork of an older NBT Editor mod than IBE.

> Because the NBT-saving operation should run on the server.

Maybe for players and block objects, like IBE. But for Items it should work on client side. But doesn't.

I'm going back to look for an IBE replacement mod...

**Comment by qyl27** [comment]
> I landed here because I though it could replace the heavy loss of [IBE Editor](https://github.com/skyecodes/IBE-Editor) (which have dropped support since October 31st 2024). And I only noticed after that this is a fork of an older NBT Editor mod than IBE.
> 
> > Because the NBT-saving operation should run on the server.
> 
> Maybe for players and block objects, like IBE. But for Items it should work on client side. But doesn't.
> 
> I'm going back to look for an IBE replacement mod...

Yes, it is designed for professional users like modders and map creators, not for players.

**Comment by nooksmr** [comment]
Im a server owner, not a player. Can't make it work in MP anyways so FFs.

**Comment by qyl27** [comment]
> Im a server owner, not a player. Can't make it work in MP anyways so FFs.

Yes, it couldn't be used unless you have installed it on the server.
